vcf/parser.py

```diff
@@ -641,7 +638,7 @@ class Writer(object):
     """VCF Writer. On Windows Python 2, open stream with 'wb'."""
 
     # Reverse keys and values in header field count dictionary
-    counts = dict((v,k) for k,v in field_counts.iteritems())
+    counts = dict((v, k) for k, v in list(field_counts.items()))
```

Comment:
The call to list is probably unnecessary and the more natural syntax is braces.
```suggestion
    counts = {v: k for k, v in field_counts.items()}
```